app/posts/apply-for-teacher-training/2021-02-10-work-history-iteration-post-testing.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Work history iteration
+description: Further changes to the work history section made after user testing
+date: 2021-01-10
+related:
+  items:
+  - text: Work history breaks
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history-breaks/
+    description: Previous post about breaks in work history
+  - text: Work history iteration for testing
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history-iteration-for-testing/
+    description: Changes to the work history flow made for testing
+---
+
+We made some further changes to the Work history section in response to user testing.
+
+## What we did
+
+We moved the section back underneath a Work experience section, and reinstated the Unpaid experience section.
```

Comment:
Think the _a_ after _underneath_ should be _the_

Reply:
👍